### ISSUE TITLE
RDL UDP in top-level component is silently dropped in XML output

### ISSUE THREAD

**Issue by amykyta3** [Juniper/open-register-design-tool#47]
Given the following RDL input:
```
property my_rdl_udp {
    type=boolean;
    component=addrmap | regfile;
};

addrmap my_top_addrmap {
    my_rdl_udp;
    reg {} a;
} top;
```

The entry for the user-defined property is missing from XML export. This seems to only occur if the property assignment occurs in the top-level component.
The following XML I would expect is missing:
```
<user_properties>
  <my_rdl_udp>true</my_rdl_udp>
</user_properties>
```

**Comment by sdnellen** [comment]
fix committed in 180406.01. will be picked up in next rls.

**Comment by amykyta3** [comment]
Confirmed fix in pre-release 180502.01
